src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      id: study_init
+      author: jhert
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(1000)
+                  constraints:
+                    nullable: false
```

Comment:
Should we put a unique constraint on the study name?

Reply:
I think the column names should be made much shorter. Postgres limits identifiers to 63 characters by default. It can be increased in the source code. It is not configurable by us for Cloud SQL.

SQL Server and Oracle limit is 128. MySQL is 64 except for alias that are 255. What do you think?

Reply:
63 should be fine as long as we add a validation step to check this on study creation. I will add this to the validation section of the doc.

Reply:
Study name seems like it should be unique--though we should make sure that when studies are removed they have been fully cleaned up and re-use of their name wont cause weird behavior (maybe more likely in the UI?)